tests/integration/test_smoke.py

```diff
@@ -115,12 +116,18 @@
 }
 
 
-def _aws(command_string):
+def _aws(command_string, max_attempts=1, delay=5, expected_rc=0):
```

Comment:
I wonder if there is a better name we can use for `expected_rc`? Usually, I expect that it is making an assertion if an `expected_rc` is provided, but the value is being used to determine if we need a retry. So maybe something like `target_rc` or `desired_rc` that indicates we will stop making attempts until we hit that rc?

Reply:
sure